README.md

```diff
@@ -1,3 +1,7 @@
+## ⛔ [DEPRECATED] This addon is no longer maintained and is now considered deprecated in favour of Storybook's native [nesting](https://storybook.js.org/docs/react/get-started/whats-a-story#nesting-stories) and [component hierarchy](https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy). We recommend Storybook version 6 be used instead.
```

Comment:
Since the nesting stories link doesn't go anywhere anymore, maybe just link both `[nesting and component hierarchy]` with the second link?